lib/inferno/apps/web/serializers/test_group.rb

```diff
@@ -7,7 +7,8 @@ class TestGroup < Serializer
         field :title
         field :description
         field :test_count
-        # field :run_as_group
+        field :run_as_group?, name: :run_as_group
+        field :parent_run_as_group?, name: :parent_run_as_group
```

Comment:
I'd be interested in hearing your and rob's opinions on just replacing both of these with a boolean `runnable` field.

Reply:
I like that, but maybe `can_be_run` instead to avoid confusion with runnables.

Reply:
I ended  up just removing the `run_as_group` field because it wasn't necessary. Now I just use the `parent_run_as_group` field as a not runnable indicator.

Reply:
`parent_run_as_group` seems like a very awkward field to include in a public api. Why would this entity contain information about its parent rather than about itself?